Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -97,6 +100,10 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
 
   override func viewWillAppear(_ animated: Bool) {
     super.viewWillAppear(animated)
+    self.locationManager.delegate = self
+    self.locationManager.desiredAccuracy = kCLLocationAccuracyBest
```

Comment:
This is really just to get the country code so we don't really need something super precise. Though we value the battery usage so a good compromise would be `kCLLocationAccuracyKilometer` because WiFi based location is enough (from the [Apple doc](https://developer.apple.com/documentation/corelocation/cllocationmanager)):

> For the services you use, configure any properties associated with that service accurately. Core Location manages power aggressively by turning off hardware when it is not needed. For example, setting the desired accuracy for location events to one kilometer gives the location manager the flexibility to turn off GPS hardware and rely solely on the WiFi or cell radios, which can lead to significant power savings.

Reply:
Updated